docs/user/FitTutorial.rst

```diff
@@ -0,0 +1,279 @@
+    # as presented, but it's defined over ages and times.
+    mortality = siler_default()
+    print(f"Mortality as a function of age, "
+          f"at 0 {mortality(0):.4f} and at 20 {mortality(20):.4f}")
+
+
+.. parsed-literal::
+
+    Mortality as a function of age, at 0 0.2032 and at 20 0.0045
+
+
+Dismod-AT is a random effects model where the micro-level is the data
+and the macro-level is the location where that data was observed.
+Dismod-AT assumes the locations form a hierarchy and that, for any fit
+or predict, there is one parent location. All data measured within the
+children of that parent location will be included in the fit.
+
+Specify locations as a Pandas DataFrame with columns for the name,
+location id as an integer, and the parent location as an integer. The
+name is purely cosmetic.
+
+.. code:: python
+
+    parent_location = 1
+    locations = pd.DataFrame(dict(
+        name=["global"],
+        parent_id=[nan],
+        location_id=[parent_location],
+    ))
+
+Make Data Using Dismod-AT Predict
+---------------------------------
+
+The siler distribution is the theoretically-known rate. Use the
+Dismod-AT predict function to create integrands such as prevalence,
+with-condition, or mtother, from this base rate.
+
+The input to Dismod-AT Predict is a representation of a continuous rate
+as an interpolation on a grid. Make a grid that has yearly age points
+and is defined for one year in time. The way Dismod-AT handles
+interpolation, all times outside the bounds of the grid points are
+constant at the nearest grid point value, so this means the rate is
+constant in time across all times, but still varies by age.
+
+.. code:: python
+
+    age_points = 121
+    omega = Var(np.linspace(0, 120, age_points), [2000])
+    for age, time in omega.age_time():
+        omega[age, time] = mortality(age)
+
+While this model has only one rate, Dismod-AT models can have five
+rates. We put this rate into a larger structure, called a
+``DismodGroups``, so that it knows which rate we're giving it.
+
+.. code:: python
+
+    model_variables = DismodGroups()
+    model_variables.rate["omega"] = omega
+
+Dismod-AT predict calculates values of integrands at different ages and
+times. We need to tell it what we want it to calculate. This looks a lot
+like how we specify data to a Dismod-AT fit. This example asks for the
+fraction of susceptibles and for mtother, which should be exactly the
+rate we put in as omega, given that the lower and upper ages are the
+same and lower and upper times are the same.
+
+.. code:: python
+
+    data_cnt = 60
+    avgints = pd.DataFrame(dict(
+        integrand="susceptible",
+        location=parent_location,
+        age_lower=np.linspace(0, 120, data_cnt),
+        age_upper=np.linspace(0, 120, data_cnt),
+        time_lower=2000,
+        time_upper=2000,
+    ))
+    avgints = pd.concat([avgints, avgints.assign(integrand="mtother")])
```

Comment:
This is concise but unclear. For a tutorial I would explicitly build out both the susceptible and mtother dataframes or try to come up with something equally obvious.